docs/docs/atom-archive/getting-started/sections/atom-basics.md

```diff
@@ -0,0 +1,331 @@
+---
+title: Atom Basics
+---
+
+::: danger STOP
+This is being kept for archival purposes only from the original Atom documentation. As this may no longer be relevant to Pulsar, you use this at your own risk.
+Current Pulsar documentation is found at [Generic Link](link/to/main/docs/section).
```

Comment:
```suggestion
Current Pulsar documentation is found at at the [documentation home](/docs/).
```

Should probably remove my `Generic Link` and replace with something like [Documentation](/docs/). This one was my fault...